dotnet/samples/GettingStartedWithProcesses/Models/MarketingNewEntryDetails.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Models;
+
+/// <summary>
+/// Holds details for a new entry in a marketing database, including the account identifier, contact name, phone number, and email address.<br/>
+/// Model used in Step02_AccountOpening.cs samples
+/// </summary>
+public class MarketingNewEntryDetails
```

Comment:
We we sure this model (and others) is intended to be _mutable_, if not I might consider `init` instead of `set`)...or even a `record`:

```c#
public record MarketingNewEntryDetails(Guid AccountId, string Name, string PhoneNumber, string Email);
```

Reply:
addressed in latest iteration